pom.xml

```diff
@@ -193,6 +193,18 @@
                     </execution>
                 </executions>
             </plugin>
+			<plugin>
+                <inherited>true</inherited>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-compiler-plugin</artifactId>
+                <version>3.0</version>
```

Comment:
Leaving version as is, as we have 3.0 in rest of the pom's.  We can update to latest plugin in all pom files